src/Config.cpp

```diff
@@ -404,8 +419,30 @@ void readConfig()
             break;
 #endif
 
+#if MF_CUSTOMDEVICE_SUPPORT == 1
+        case kTypeCustomDevice:
+            adrType      = addreeprom; // first location of custom Type in EEPROM
+            copy_success = readEndCommandFromEEPROM(&addreeprom, '.');
+            if (!copy_success)
+                break;
+
+            adrPin       = addreeprom; // first location of custom pins in EEPROM
+            copy_success = readEndCommandFromEEPROM(&addreeprom, '.');
+            if (!copy_success)
+                break;
+
+            adrConfig    = addreeprom; // first location of custom config in EEPROM
+            copy_success = readEndCommandFromEEPROM(&addreeprom, '.');
+            if (copy_success) {
+                CustomDevice::Add(adrPin, adrType, adrConfig);
+                copy_success = readEndCommandFromEEPROM(&addreeprom, ':'); // check EEPROM until end of command
+            }
+            // cmdMessenger.sendCmd(kDebug, F("CustomDevice loaded"));
+            break;
+#endif
+
```

Comment:
new case for supporting custom devices. All other changes should be reformatting